### ISSUE TITLE
Any way to move my favorites into a playlist?

### ISSUE THREAD

**Issue by mhauer** [matt-h/rdio-enhancer#51]
Would need this to import it into Apple Music :)

**Comment by matt-h** [comment]
There isn't currently, but this should be possible to build. Is this because Apple Music is able to import a playlist but not your favorites?

**Comment by mhauer** [comment]
Yes, I'm using a tool which can import playlists to Apple Music, but not my Favorites. Unfortunately.
